solution.cpp

```diff
@@ -1,23 +1,137 @@
-/* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+/* -------------------------------------------------------------------------*
+* Copyright &copy; 2015 Paul Sihavong paul.sihavong@hotmail.com				*
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)		*
+* ------------------------------------------------------------------------ */
 
-/**
- * TODO: program description
+/*Program Description*
+ *This program is meant to convert the user input of an integer from 1 to 4000 to a
+ *roman numeral. Assuming the user gives an input within a correct parameter, the program
+ *will convert the integer to the correct roman numeral.
  */
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
 
-// ----------------------------------------------------------------------------
+using std::cin;
+using std::cout;
+using std::endl;
+using std::string;
 
-int main() {
-    // TODO
+string romanNumeralConverter(int a, int c);
 
-    return 0;  // success
+int main() {
+	int a = 0; // variable to store user input
+	int c = 0; // variable to determine what roman numerals are used
+	char d = ' '; // char to determine if the user would like to use the calculator again
+	// the start of calculator processing
+	// each numeral that is used here (e.g. M, CM, D, etc) is for the shortest way to write a roman numeral
+	// we start at the biggest roman numeral factor because we want the biggest numeral in front
```

Comment:
great documentation!  really :-) .  longer variable names would have rendered some if it unnecessary, though.